tasks/main.yml

```diff
@@ -111,37 +123,43 @@
     creates: /usr/local/ssl/private/dhparams.pem
 
 - name: Stat the default site SSL cert
-  shell: "test -e /etc/letsencrypt/live/{{ default_site_fqdn }}"
+  stat:
+    path: "/etc/letsencrypt/live/{{ default_site_fqdn }}"
   register: default_cert_result
-  changed_when: "default_cert_result.rc != 0"
-  ignore_errors: true
-
+- debug:
+    var: default_cert_result
 - name: Compose the certbot command string for the default site cert
   set_fact:
     certbot_command_string: >-
-      /usr/local/bin/certbot-auto certonly
-      --non-interactive
-      --agree-tos
-      --email {{ default_mail_recipient }}
-      --webroot
-      --webroot-path {{ letsencrypt_webroot }}
-      --domains {{ default_site_fqdn }}
+      certbot --nginx --email {{ letsencrypt_expiry_email }} --agree-tos --non-interactive -d {{ default_site_fqdn }}
 
 - name: Create a cert for the default site (can take some time)
   shell: >
-    {{ certbot_command_string }} --dry-run && {{ certbot_command_string }}
+    {{ certbot_command_string }}
```

Comment:
Dry run was there to prevent burning through LE's "allowed errors" quota, which locks us out from creating new certs for 7 days if crossed. Has this changed?